### ISSUE TITLE
busybox.html on webpage

### ISSUE THREAD

**Issue by jarnosz** [rmyorston/busybox-w32#242]
Request for improvement: 

Would it be possible to add a link to the busybox.html (or txt or man) file generated by the compilation to the corresponding distribution binary on `https://frippery.org/busybox/`, for a quick reference for end users? 

Thank you!

**Comment by rmyorston** [comment]
Good idea! I've edited the BusyBox.html file for the current release and have put it [here](https://frippery.org/busybox/BusyBox.html). There's a link to it from my web page.

**Comment by jarnosz** [comment]
Thank you!

**Comment by jarnosz** [comment]
Wait! Where in `https://frippery.org/busybox/index.html` is the link?

**Comment by rmyorston** [comment]
>Where in https://frippery.org/busybox/index.html is the link?

It's at the end of the paragraph that begins 'Other staples of Unix shell programming...'.
